### PR TITLE
Fixed compatibility with new macOS, updated to 6.0 aapt (+ changes to…

### PR DESCRIPTION
**Pull request by AuroraWright**
… make it work) so that new apks work (7.x aapts have bugged labels), changed output layout slightly, added SDK/target SDK display, fixed icon being attempted to be shown when there is none

### CODE REVIEW

**Comment by hezi** [comment]
Hi! Thank you for the contributing :) 

What aapt version is the new one you bundled, where did you get it from?
I'm thinking about not bundling any binaries with this at all and just fetching them from a reputable source during build... 
 
What were the changes that made it work with newer macOS versions? (I'm guessing giving it an actual `LSItemContentTypes`?)

Thanks! 🎉

**Comment by AuroraWright** [comment]
Hi, I got that aapt from the Google build tools (the latest API 23/Marshmallow build). Any subsequent versions are bugged and display a wrong or empty label (https://code.google.com/p/android/issues/detail?id=213397).
Yeah setting a proper LSItemContentType fixed it on Sierra at least.

**Comment by AuroraWright** [comment]
The plugin broke on me after switching from Entropy to BetterZip, so I took a look at LaunchServices, and it appears that content type was declared by Entropy, while BetterZip merges .apks in the zip content type. I tried copying the declaration from Entropy to the qlgenerator itself, but it appears in recent versions of macOS Apple only allows to do that within apps.
tl;dr there's no surefire way to have this work on latest OS versions, what I did is hacking the declaration into the Info.plist of Android File Transfer.
